enterprise/api/v1/views/enterprise_customer.py

```diff
@@ -66,6 +64,22 @@ class EnterpriseCustomerViewSet(EnterpriseReadWriteModelViewSet):
     filterset_fields = FIELDS
     ordering_fields = FIELDS
 
+    def get_queryset(self):
+        """
+        Allow PAs to access all enterprise customers
+        """
+        queryset = self.queryset
+        if self.action in ('create', 'partial_update', 'update', 'retrieve', 'list') and \
+                self.request.user.groups.filter(name=PROVISIONING_ADMINS_GROUP).exists():
+            # allow PAs to access all enterprise customers
+            self.filter_backends = (
+                filters.OrderingFilter, DjangoFilterBackend)
+            return models.EnterpriseCustomer.active_customers.all()
```

Comment:
Yes it returns queryset ... [Ref](https://github.com/openedx/edx-enterprise/blob/4a29013429c93070e47e8156e1519b73d1473eb3/enterprise/api/v1/views/enterprise_customer.py#L56)

But since it's already given in class..I've refactored it now!